packages/lib-sourcify/src/lib/types.ts

```diff
@@ -540,3 +540,9 @@ export interface CompiledContractArtifacts {
   creationBytecodeCborAuxdata: CompiledContractArtifactsCborAuxdata;
   runtimeBytecodeCborAuxdata: CompiledContractArtifactsCborAuxdata;
 }
+
+export interface AuxdataDiff {
```

Comment:
Adding the type
- Removed offsetEnd because that's implicit from diffStart + diff.length. Also it's possible to use just real.length

Reply:
I wanted to make sure to rebuild the original auxdata only using the information from this object, but it actually makes no sense